docs/versioned_docs/version-7.4.x/behaviour.md

```diff
@@ -3,7 +3,7 @@ id: behaviour
 title: Behaviour
 ---
 
-1. Any request passing through the proxy (and not matched by `--skip-auth-regex`) is checked for the proxy's session cookie (`--cookie-name`) (or, if allowed, a JWT token - see `--skip-jwt-bearer-tokens`).
+1. Any request passing through the proxy (and not matched by `--skip-auth-route`) is checked for the proxy's session cookie (`--cookie-name`) (or, if allowed, a JWT token - see `--skip-jwt-bearer-tokens`).
```

Comment:
Can you update the same file in `docs/docs` too else this will revert on the next release